src/components/sbb-tooltip-trigger/sbb-tooltip-trigger.tsx

```diff
@@ -46,7 +50,7 @@ export class SbbTooltipTrigger implements ComponentInterface, ButtonProperties {
 
     return (
       <Host {...hostAttributes}>
-        <span class="sbb-tooltip-trigger">
+        <span class="sbb-tooltip-trigger" data-icon-small={this._isInFormField}>
```

Comment:
The icon-button mixin wouldn't work if I did that because it uses :host() selectors. To separate the cases where the tooltip trigger is inside a form field from the cases where it isn't, I should use something like :host([data-icon-small]) { @include sbb.icon-button }, using essentially a host selector inside another host selector, causing the mixin to have no effect.

Reply:
I see the problem which let me have a deeper look at the mixin. In order to support also the mixin being included in the sbb-button, I think we should refactor it the way we often do it:
- split up the mixins into xy--variables where there are the default variables contained
- create mixins with only variables for every state (hover, active, focus, ..?) without containing any selectors. focus visible will probably need one mixin more.
- define the selectors only in the consuming component